dotnet/src/MicroFocus.CafApi.QueryBuilder.Vqll.Builder.Internal/IJsonBuilder.cs

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace MicroFocus.CafApi.QueryBuilder.Vqll.Builders.Internal
+{
+    public interface IJsonBuilder
+    {
+        void WriteBoolean(bool value);
+
+        void WriteEndArray();
+
+        void WriteNull();
+
+        void WriteNumber(long? v);
```

Comment:
Why would we be making this nullable?

```suggestion
        void WriteNumber(long v);
```